packages/fxa-content-server/tests/server/routes/validate-email-domain.js

```diff
@@ -57,7 +57,17 @@ registerSuite('routes/validate-email-domain', {
           assert.isTrue(res.json.calledOnceWith({ result: 'MX' }));
         });
       },
-
+      'responds with result {result: "skip"} when there is a skip record}': () => {
+        const validateEmailDomainRoute = require('../../../server/lib/routes/validate-email-domain');
+        const req = { query: { domain: 'skip.abc.xyz' } };
+        const res = { json: sinon.stub() };
+        const route = validateEmailDomainRoute({
```

Comment:
@chenba  Here what I will do I will call `config` object from `../../../server/lib/configuration.js` and pass the config object to the route.